packages/flutter/test/widgets/form_test.dart

```diff
@@ -1126,4 +1126,174 @@ void main() {
     await tester.pump();
     expect(find.text('test_error'), findsNothing);
   });
+
+  testWidgets('AutovalidateMode.onUnfocus', (WidgetTester tester) async {
+    final GlobalKey<FormState> formKey = GlobalKey<FormState>();
+    String? errorText(String? value) => '$value/error';
+
+    Widget builder() {
+      return MaterialApp(
+        theme: ThemeData(),
+        home: MediaQuery(
+          data: const MediaQueryData(),
+          child: Directionality(
+            textDirection: TextDirection.ltr,
+            child: Center(
+              child: Form(
+                key: formKey,
+                autovalidateMode: AutovalidateMode.onUnfocus,
+                child: Material(
+                  child: Column(
+                    children: <Widget>[
+                      TextFormField(
+                        initialValue: 'bar',
+                        validator: errorText,
+                      ),
+                      TextFormField(
+                        initialValue: 'bar',
+                        validator: errorText,
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    await tester.pumpWidget(builder());
+
+    // No error text is visible yet.
+    expect(find.text(errorText('foo')!), findsNothing);
+
+    // Enter text in the first TextFormField.
+    await tester.enterText(find.byType(TextFormField).first, 'foo');
+    await tester.pumpAndSettle();
+
+    // No error text is visible yet.
+    expect(find.text(errorText('foo')!), findsNothing);
+
+    // Tap on the second TextFormField to trigger validation.
+    await tester.tap(find.byType(TextFormField).last);
+    await tester.pumpAndSettle();
```

Comment:
Can you expect the presence of the two errorTexts here after the first tap as well?

Reply:
Not in this case. Only if I actually clicked outside the Form.